### PR TITLE
add homework 14

### PR DESCRIPTION
**Pull request by superpro2007**
used psycopg2

### CODE REVIEW

File: homework_14/.gitignore
Comment (nicilytop-tms) [inline homework_14/.gitignore:1]: dont need it on each folder.. its enough to add gitignore on root directory

Comment (nicilytop-tms) [reply homework_14/.gitignore:1]: lets remove it

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]
Nice job except additional .gitignore 😎

File: homework_14/.gitignore
Comment (superpro2007) [reply homework_14/.gitignore:1]: > lets remove it

done thx

**Review by nicilytop-tms — APPROVED** [review]

**Pull request merged**